workers/README_QUANT.md

```diff
@@ -0,0 +1,41 @@
+# refine.bio RNA-Seq Sample Compendium
+
+This is a refine.bio RNA-seq sample compendium.
+It represents the Salmon output for the collection of RNA-seq samples from an organism that can be processed with refine.bio.
+Each individual sample has its own `quant.sf` file; the samples have not been aggregated and normalized.
+RNA-seq sample compendia are designed to allow users that are comfortable handling these files to generate output that is most useful for their downstream applications.
+
+Please see the [Salmon documentation on the `quant.sf` output format](https://salmon.readthedocs.io/en/latest/file_formats.html#quantification-file) for more information.
+
+## Contents
+
+This download includes individual `quant.sf` files that that are organized into to folders that represent experiments.
+The individual files represent individual runs (e.g., SRR, ERR, DRR) and the folders represent projects (e.g., SRP, ERP, DRP) from Sequence Read Archive.
+We also included any metadata available from refine.bio, which is limited for RNA-seq data at this time.
+
+* The `aggregated_metadata.json` file contains experiment (project) metadata.
+
+* Each folder corresponds to an experiment and contains a metadata TSV file and `quant.sf` files (named following the convention `<sample-accession-id>_quant.sf`) for each sample that refine.bio was able to process with Salmon.
+
+* Sample metadata (e.g. disease vs. control labels) are contained in the TSV file with `metadata` in the filename as well as any JSON files.
+We apply light harmonization to some sample metadata fields, which are denoted by `refinebio_` (`refinebio_annotations` is an exception).
+The contents of a sample's `refinebio_annotations` field in JSON files include the submitter-supplied sample metadata.
+
+* Experiment metadata (e.g., experiment title and description) are contained in JSON files with `metadata` in the filename.
+
+_Note that we use the terms "sample" and "experiment" to be consistent with the rest of refine.bio, but files will use run identifiers and project identifiers, respectively._
+
+Please see [our documentation](http://docs.refine.bio/en/latest/) for more details.
```

Comment:
Could we update this with the link to the RNA-seq Sample Compendia section once we have it?